lib/routes/espn/news.ts

```diff
@@ -0,0 +1,82 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+
+export const route: Route = {
+    path: '/news/:sport',
+    name: 'News',
+    maintainers: ['GymRat102'],
+    example: '/news/nba',
+    categories: ['traditional-media'],
+    parameters: { sport: 'sport category, can be nba, nfl, mlb, nhl etc.' },
+    description: `Get the news feed of the sport you love on ESPN.
+| Sport                |  sport  |  Sport         |  sport  |
+|----------------------|---------|----------------|---------|
+| 🏀NBA                | nba     | 🎾Tennis       | tennis  |
+| 🏀WNBA               | wnba    | ⛳️Golf         | golf    |
+| 🏈NFL                | nfl     | 🏏Cricket      | cricket |
+| ⚾️MLB                | mlb     | ⚽️Soccer       | soccer  |
+| 🏒NHL                | nhl     | 🏎️F1           | f1      |
+| ⛹️College Basketball | ncb      | 🥊MMA          | mma     |
+| 🏟️️College Football   | ncf     | 🏈UFL          | ufl     |
+| 🏉Rugby              | rugby   | 🃏Poker        | poker   |`,
+    radar: [
+        {
+            source: ['espn.com/:sport*'],
+            target: '/news/:sport',
+        },
+    ],
+    handler: async (ctx) => {
+        const { sport = 'nba' } = ctx.req.param();
+        const response = await ofetch(`https://onefeed.fan.api.espn.com/apis/v3/cached/contentEngine/oneFeed/leagues/${sport}?offset=0`, {
+            headers: {
+                accept: 'application/json',
+            },
+        });
+
+        const handledTypes = new Set(['HeadlineNews', 'Story', 'Media', 'Shortstop']);
+        const list = response.feed
+            .filter((item) => handledTypes.has(item.data.now[0].type))
+            .map((item) => {
+                const itemDetail = item.data.now[0];
+                const itemType = itemDetail.type;
+
+                return {
+                    // distinguish among normal news/stories, videos and shortstops
+                    title: itemType === 'Media' ? `[Video] ${itemDetail.headline}` : itemType === 'Shortstop' ? `[Shortstop] ${itemDetail.headline}` : itemDetail.headline,
```

Comment:
```suggestion
                    title: itemDetail.headline,
```
Don't change the title